larpandoracontent/LArObjects/LArMCParticle.h

```diff
@@ -21,13 +21,46 @@
 namespace lar_content
 {
 
+// ATTN - Enumeration replicates the numbering in the CellTree module and thus has a dummy UNUSED enumeration at 15 to account for an used
+// duplication in the CellTree process map
+// We also add an INCIDENT_NU to assign a process for neutrino MC given that we don't have a distinct class for neutrino MC
```

Comment:
Wonder if this comment can be made a bit more self-explanatory, and give slightly more general context? Aim to keep very brief and easy to maintain in future, of course (comments otherwise can "go off"!).